src/client/mod.rs

```diff
@@ -85,9 +86,9 @@ impl<C, B> Client<C, B> {
     fn configured(config: Config<C, B>, exec: Exec) -> Client<C, B> {
         Client {
             connector: Arc::new(config.connector),
-            executor: exec,
+            executor: exec.clone(),
```

Comment:
We should be able to completely remove the executor field of the client, and always use `cx`.

(`HttpConnector` should still own its own executor, but that's because it needs to spawn blocking tasks. It should be enough to allow configuring the `HttpConnector` manually.)

Reply:
- [ ] remove custom executor